packages/notebook-extension/src/index.ts

```diff
@@ -478,12 +478,65 @@ const trusted: JupyterFrontEndPlugin<void> = {
   },
 };
 
+/**
+ * Add a command to open right sidebar for Editing Notebook Metadata when clicking on "Edit Notebook Metadata" under Edit menu
+ */
+const editNotebookMetadata: JupyterFrontEndPlugin<void> = {
+  id: '@jupyter-notebook/notebook-extension:edit-notebook-metadata',
```

Comment:
@Dilip-Jain curious if you tried adding the command to the menu settings, as mentioned in https://github.com/jupyter/notebook/issues/6402#issuecomment-1743087342 ?

To know if the plugin is necessary, or if the menu entry would be enough.

Reply:
For enhancing the "Edit Notebook Metadata" feature, adhering to the user experience, the objective was to open both the right sidebar (showing the "Notebook Tools" widget) and automatically expand the "Additional Tools" collapsible section, which houses the notebook metadata editor. This streamlined action ensures a smoother and more intuitive user interaction.

The desired action involves two key steps:

1. **Opening the Right Sidebar:** This step involves displaying the "Notebook Tools" widget, which is housed in the right sidebar.
2. **Expanding the "Additional Tools" Section:** Within the right sidebar, the "Additional Tools" collapsible section contains the notebook metadata editor. This section needs to be automatically expanded to provide immediate access to the editor.

The challenge faced was that using the existing command `application:toggle-panel` with specified arguments in the menus.json settings file accomplished the first step—opening the right sidebar. However, it did not automatically expand the "Additional Tools" section to reveal the notebook metadata editor so to overcome this limitation a custom plugin was added.

Screenshot: Right Sidebar panel:
**Using command menu entry:**
![image](https://github.com/jupyter/notebook/assets/59838651/96835645-9fec-43dd-bb73-7f6b4e16ebcd)

**Using the plugin:**
![image](https://github.com/jupyter/notebook/assets/59838651/cff0100b-e99a-41ae-829e-c3e55df44935)